app/stores/skiplagged.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+const commonPokemon = require(__appbase + 'models/pokemonSighting'),
+      async = require('async'),
+      config = require(__base + 'config');
+
+module.exports = {
+    /*
+     * inserting the pokemon details
+     */
+    add: function (data, callback) {
+        var pokemon = new commonPokemon();
```

Comment:
This whole file looks the same as pokeradar(except for the config values), so just a single file can be used and the collection name can be accepted as a parameter to distinguish.